cunumeric/deferred.py

```diff
@@ -3334,9 +3394,10 @@ def unpackbits(
         task.execute()
 
     @auto_convert([1])
-    def _wrap(self, src: Any, new_len: int) -> None:
+    def _wrap(self, src: DeferredArray, new_len: int) -> None:
+        src = self.runtime.to_deferred_array(src)
```

Comment:
@manopapad : I had to explicitly convert src to deferred array in the case when src in a scalar array . I thought auto_convert[1] should have taken care of this. Any thoughts ?

Reply:
https://github.com/nv-legate/cunumeric/pull/640 fixes this (you don't need to call `to_deferred_array` manually). If you don't want to wait for that, then you can pass the arguments as follows in `array.py::_wrap`:

```
        out._thunk._wrap(self._thunk, new_len=new_len)
```

Reply:
Thank you, Manolis!

Reply:
I will wait until #640 is merged

Reply:
I merged the fix and updated the code accordingly.